README.md

```diff
@@ -46,6 +46,22 @@ The app can be run using Vagrant
 $ vagrant up
 ```
 
+OR
+
+The app can be run using Docker 
+```bash
+$ docker build --tag todo-app .
+$ docker run -it --rm --name=todo -p 80:5000 -d todo-app
```

Comment:
Assuming this for development, where is the bind mount to local files (for hot reloading)?

Reply:
bind mount was held in the makefile. ive updated the readme to have the full docker build scripts